cmd/storj-admin/template/template.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+//go:generate $GOPATH/bin/go-bindata -pkg template -ignore template.go -ignore bindata.go .
+
+package template
+
+import (
+	"html/template"
+)
+
+var (
+	Public = template.Must(
+		template.New("public").Parse(string(MustAsset("public.html.tmpl"))),
```

Comment:
There's only one file, you can write html in Go code directly.

Reply:
I am using this route as i plan on splitting the pages into subtemplates to make them better scoped.